docs/app/content-layouts.md

```diff
@@ -0,0 +1,258 @@
+</video>
+
+This Layout is ideal for Collections with time-oriented data _(e.g. events and appointments)_. It includes the following
+controls.
+
+**Layout Options**
+
+- **Display Template** — Set a mix of Field values and text to represent Items on the calendar.
+- **Start Date Field** — Field to determine each Item's beginning time on the calendar.
+- **End Date Field** — Field to determine each Item's ending time on the calendar.
+
+**Subheader**
+
+- **Toggle Month and Year** — Move across time with the chevrons in the subheader.
+- **Today** — Click to jump to the current date on the calendar.
+- **Month Week Day List** — Adjust the time interval used to display Items in the Page Area.
+
+**Page Area**
+
+- **Select Item** — Click an Item on the calendar to open its Item Page.
+
+:::tip Configuration Requirements
+
+To use this, the Collection will need at least one datetime Field to set a start time, but ideally two _(to set a start
+and end time)_.
+
+:::
+
+## Map Layout
+
+<video autoplay muted loop controls title="The Map Layout">
+	<source src="" type="video/mp4" />
+</video>
+
+This Layout is ideal for Collections that emphasize geospatial data. It provides a world map, with Items displayed as
+points, lines, and other geometry. The following options are available:
+
+**Layout Options**
+
+- **Basemap** — Choose the map provider used for the Collection.
+- **Geospatial Field** — Select the geospatial field type to display on screen.
+  - **Map JSON Point** — Supports latitude-longitude based, single-point locations.
+  - **Map Geometry** — Supports geometric areas such as lines and polygons.
+- **Display Template** — Choose the Fields displayed when hovering over an Item on the map.
+- **Cluster Nearby Data** — Toggle whether or not nearby Items get clustered into a single pin.
+
+**Subheader**\
+There is no Subheader on the Map Layout.
+
+**Page Area**
+
+- **Zoom** — Click <span mi icon>add</span> and <span mi icon>remove</span> in the upper left hand corner of the Page
+  Area to zoom in and out.
+- **Zoom to my Location** — Click <span mi icon>my_location</span> to zoom into your current location on the map.
+- **Reframe** — Click the square in the upper left-hand corner to resize.
+- **Select Item** — Click a single Item to enter the Item Page.
+- **Select Items** — Click and drag to select multiple Items at once to open the Item Page.
+
+:::tip Configuration Requirements
+
+Must have a Map Field configured on the Collection for this Layout to work properly.
+
+:::
+
+## Kanban Layout
+
+<video autoplay muted loop controls title="The Kanban Layout">
+	<source src="" type="video/mp4" />
+</video>
+
+This Layout is ideal for Collections that serve as project management tools or to-do lists, where each Item represents a
+task. Tasks are grouped onto status panels. The following controls are available.
+
+**Layout Options**
+
+- **Group By** — Select the Field used to define Item status. See below for details.
```

Comment:
How to name the kanban boards? 

Group By vs Status Panels vs kanban boards vs etc???